SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -142,25 +158,20 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 		QueueOutgoingUpdate(Channel, Actor, HandleUnresolvedObjectsPair.Key, HandleUnresolvedObjectsPair.Value, /* bIsHandover */ true);
 	}
 
-	for (int RPCType = 0; RPCType < RPC_Count; RPCType++)
+	for (int32 RPCType = TYPE_ClientRPC; RPCType < TYPE_Count; RPCType++)
```

Comment:
Omega-nit: we could have the enum values for the first and last RPC type instead of relying on the ClientRPC to be the first one and no schema types being after the RPCs.